ipfssync/ipfssync.go

```diff
@@ -20,10 +20,15 @@ import (
 	"google.golang.org/protobuf/proto"
 
 	"go.vocdoni.io/dvote/data"
-	"go.vocdoni.io/dvote/log"
+	logger "go.vocdoni.io/dvote/log"
 	"go.vocdoni.io/dvote/statedblegacy/gravitonstate"
 )
 
+// global log is set to a logger.Named(loggerName) on Start
+var log = logger.Logger()
+
+const loggerName = "ipfssync"
```

Comment:
why not just `var log = logger.Named("ipfssync")`?

Reply:
Named() clones the global logger, which at the time of package init is uninitialized. Afterwards, during dvotenode init() , log.Init() is called and sets the global logger up, but it's too late.
i'll see how i can refactor Named() so that the when log.Init() is later run, the named loggers are also init'd

Reply:
I think the right solution is to instead initialize the log package's global logger at init time as well. then the settings applied via flags would modify the logger in-place.

Reply:
right, i did this, adapting code from `ipfs/go-log` which had exactly this approach.
it was an interesting adventure :)